manifests/a/Autodesk/AutoCAD2023/2023/Autodesk.AutoCAD2023.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.9.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: Autodesk.AutoCAD2023

```

```suggestion
PackageIdentifier: Autodesk.AutoCAD.2023
```
After changing the package identifier, you will also need to change the path of the manifests to `./manifests/a/Autodesk/AutoCAD/2023/24.2.53.0/*`